.env.example

```
@@ -9,3 +9,6 @@ PINATA_API_KEY=123...
 PINATA_SECRET_API_KEY=123...
 SERVICE_EVENTS=0
 SERVICE_ARWEAVE=0
+PUSHER_BEAMS_INSTANCE_ID=
+PUSHER_BEAMS_SECRET_KEY=
```

Can you add a prefix `SERVICE_` to these vars? SERVICE prefix mean the feature is optional. 
```suggestion
SERVICE_PUSH_NOTIFICATIONS=0
SERVICE_PUSHER_BEAMS_INSTANCE_ID=
SERVICE_PUSHER_BEAMS_SECRET_KEY=
```